test/itemrenderer.cpp

```diff
@@ -7,6 +7,16 @@
 
 using namespace newsboat;
 
+static const auto FEED_TITLE = std::string("Funniest jokes ever");
+std::shared_ptr<RssFeed> test_feed(Cache* c)
```

Comment:
Cosmetic: I would call it something like `make_feed` or `make_test_feed`, as it creates feed, not test it.

(Yeah, you can deduce that from the signature, but still...).

Reply:
Good point! I renamed `test_feed` to `create_test_feed`, and `test_item` to `create_test_item`. The latter is a bit misleading since it creates both an item and a feed, but that's obvious from the usage as we always unpack the resulting `std::pair` into local variables.